src/components/Assets/AssetsList.tsx

```diff
@@ -68,32 +66,31 @@ const AssetsList = () => {
       qParams.warranty_amc_end_of_validity_after || "",
   };
 
-  const { refetch: assetsFetch, loading } = useTanStackQueryInstead(
-    routes.listAssets,
-    {
-      query: params,
-      onResponse: ({ res, data }) => {
-        if (res?.status === 200 && data) {
-          setAssets(data.results);
-          setTotalCount(data.count);
-        }
-      },
-    },
-  );
+  const {
+    refetch: assetsFetch,
+    isLoading: loading,
+    data: assetList,
```

Comment:
asset list is incorrect wording. data is an object and not a list. data.results is a list.

also violates naming conventions: ref: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variable_names

<img width="874" alt="image" src="https://github.com/user-attachments/assets/e77c704f-0d45-4c55-b473-4817a8a5c445" />


```suggestion
    data,
```